app/authoring/user-interactions.md

```diff
@@ -23,13 +23,19 @@ The `prompt` method is asynchronous and return a promise. You'll need to return
 ```js
 module.exports = generators.Base.extend({
   prompting: function () {
-    return this.prompt({
+    return this.prompt([{
       type    : 'input',
       name    : 'name',
       message : 'Your project name',
       default : this.appname // Default to current folder name
-    }).then(function (answers) {
-      this.log(answers.name);
+    }, {
+      type    : 'confirm',
+      name    : 'cool',
+      message : 'Would you like to enable the Cool feature?',
+      default : this.cool
```

Comment:
As discussed with @SBoudrias, the documentation is inconsistent as it shows only a special scenario where the `prompt` array contains only one item. I extended the documentation to show a more real case in where the variable contains two questions trying to help other developers reading for the first time this tutorial